api/client/cli.go

```diff
@@ -106,6 +107,12 @@ func (cli *DockerCli) restoreTerminal(in io.Closer) error {
 	return nil
 }
 
+// AddCommonFlags installs command line flags which can control the
+// client's behavior, and which are common to all client commands.
+func (cli *DockerCli) AddCommonFlags(flags *flag.FlagSet) {
+	// nothing here yet
```

Comment:
Is implementation so complicated that we don't want it to add in this PR? I'm not a fan of such unfinished features in master :/